### ISSUE TITLE
The identification time is too long. How can I set it faster

### ISSUE THREAD

**Issue by duongnv1996** [dm77/barcodescanner#403]
It's take long time to identificate QR Code. How to set it faster?
Thank you

**Comment by govindabhattar** [comment]
Yup any resolution?

**Comment by haogg1996** [comment]
The same question

**Comment by emanuelthedude** [comment]
I have the same problem

**Comment by MikeTheWatchGuy** [comment]
How long is it taking?

**Comment by emanuelthedude** [comment]
I tested on Nexus devices (5 and 5x) and on low budged devices. It takes up to 30-40 seconds until the QR Code is scanned on any device. The QR Code is nothing special but a usual QR Code. It is printed on a credit card and has a size of 4x4cm. After a long time it seems the auto focus is starting and then the QR Code gets scanned immediately. That is why I enabled the auto focus at initialisation time but it didn't change anything.
